test/io.jl

```diff
@@ -44,17 +41,23 @@ module TestIO
                    B = 'a':'c',
                    C = ["A", "B", "C"],
                    D = CategoricalArray('a':'c'),
-                   E = NullableCategoricalArray(["A", "B", "C"]),
-                   E = NullableArray(1:3),
-                   F = NullableArray(fill(Nullable(), 3)),
-                   G = fill(Nullable(), 3))
+                   E = CategoricalArray(["A", "B", null]),
+                   F = Vector{Union{Int, Null}}(1:3),
+                   G = nulls(3),
+                   H = fill(null, 3))
 
-    answer = Sys.WORD_SIZE == 64 ? 0x3bd4a4a099ae8aac : 0x5ecdc4bb
-    @test hash(sprint(printtable, dt)) == answer
+    DRT = CategoricalArrays.DefaultRefType
+    @test sprint(printtable, dt) ==
+        """
+        "A","B","C","D","E","F","G","H"
+        1,"a","A","CategoricalArrays.CategoricalValue{Char,$DRT} 'a'","CategoricalArrays.CategoricalValue{String,$DRT} "A"","1",null,null
```

Comment:
Better use the short form of `CategoricalValue` in order to pass tests on 0.6 (cf. https://github.com/JuliaData/DataTables.jl/pull/83#discussion_r134704806). This behavior is likely going to be reverted in Julia 0.7 anyway.